elements/lisk-transactions/src/utils/validation/validation.ts

```diff
@@ -153,7 +153,7 @@ export const isNumberString = (str: string) => {
 		return false;
 	}
 
-	return /^[0-9]+$/g.test(str);
+	return /^-?[0-9]+$/g.test(str);
```

Comment:
I think isValidNumber still a misleading name. Because `123.123` would fail with this regex but it's a valid number. I was going to suggest to rename it to `isValidInteger` but seems like we already have it. So, I suggest either we would improve this to validate all numbers or remove this function?

https://www.regextester.com/96606